packages/hashi/src/iframeClient.js

```diff
@@ -41,6 +41,15 @@ export default class SandboxEnvironment {
 
     patchCrossOrigin();
 
+    let self = this;
+    window.addEventListener('DOMContentLoaded', function() {
+      const data = {
+        offsetHeight: window.document.body.offsetHeight,
+      };
+      const event = events.CONTENTLOADED;
+      self.mediator.sendMessage({ nameSpace, event: event, data: data });
+    });
```

Comment:
is this file packaged with Webpack, or is it unprocessed because it gets embedded in Hashi?

If the former, you can use an arrow function here:

```suggestion
    window.addEventListener('DOMContentLoaded', () => {
      const data = {
        offsetHeight: window.document.body.offsetHeight,
      };
      const event = events.CONTENTLOADED;
      this.mediator.sendMessage({ nameSpace, event: event, data: data });
    });
```

Reply:
It's packaged, I'll make the change! Didn't realize this got around the self = this problem.

Reply:
Yup - super convenient!

ref: https://hackernoon.com/javascript-es6-arrow-functions-and-lexical-this-f2a3e2a5e8c4